src/web/views.py

```diff
@@ -1,32 +1,44 @@
 """Views for the web app."""
-from flask import Blueprint, jsonify, render_template, request
+import shutil
+import tempfile
+from pathlib import Path
+
+from flask import Blueprint, current_app, jsonify, render_template, request
 from pydantic import ValidationError
+from web.computed_tomography import get_available_cts, validate_ct
 from web.controller import (
     get_built_in_scenarios,
+    get_default_material_properties,
     get_scenario_layout,
     get_simulation_image,
 )
 from web.messages.material_properties import MaterialName
 from web.messages.requests import RenderLayoutRequest, SimulateRequest
 from web.messages.transducers import TransducerType
 from web.simulation_runner import SimulationRunner
+from werkzeug.utils import secure_filename
 
 bp = Blueprint("main", __name__, url_prefix="/")
 
+DEFAULT_CENTER_FREQUENCY = 5e5
+
 
 @bp.route("/")
 async def index():
     """Render the index page, listing all the built-in scenarios."""
-    title = "Neurotech Web App"
     return render_template(
         "index.html",
-        title=title,
+        title="Neurotech Web App",
         has_simulation=SimulationRunner().has_last_result,
         is_running_simulation=SimulationRunner().is_running,
         configuration=SimulationRunner().configuration,
         built_in_scenarios=get_built_in_scenarios(),
         all_materials=MaterialName.get_material_titles(),
+        all_material_properties=get_default_material_properties(
+            DEFAULT_CENTER_FREQUENCY
+        ).dict(),
         all_transducer_types=TransducerType.get_transducer_titles(),
+        available_cts=get_available_cts(current_app.config["CT_FOLDER"]),
```

Comment:
When I try to open the web app in Chrome (i.e., navigating to `http://0.0.0.0:8000/`, I get the following error:
```
❯ make web
poetry run gunicorn -b 0.0.0.0 -w 1 --timeout 99999999 'src.web.app:app'
[2023-10-03 14:16:00 -0700] [64988] [INFO] Starting gunicorn 21.2.0
[2023-10-03 14:16:00 -0700] [64988] [INFO] Listening at: http://0.0.0.0:8000 (64988)
[2023-10-03 14:16:00 -0700] [64988] [INFO] Using worker: sync
[2023-10-03 14:16:00 -0700] [64991] [INFO] Booting worker with pid: 64991
Exception on / [GET]
Traceback (most recent call last):
  File "/Users/charles/mambaforge/envs/neurotechdevkit/lib/python3.10/site-packages/flask/app.py", line 2529, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/charles/mambaforge/envs/neurotechdevkit/lib/python3.10/site-packages/flask/app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/charles/mambaforge/envs/neurotechdevkit/lib/python3.10/site-packages/flask/app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/charles/mambaforge/envs/neurotechdevkit/lib/python3.10/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/charles/mambaforge/envs/neurotechdevkit/lib/python3.10/site-packages/asgiref/sync.py", line 277, in __call__
    return call_result.result()
  File "/Users/charles/mambaforge/envs/neurotechdevkit/lib/python3.10/concurrent/futures/_base.py", line 451, in result
    return self.__get_result()
  File "/Users/charles/mambaforge/envs/neurotechdevkit/lib/python3.10/concurrent/futures/_base.py", line 403, in __get_result
    raise self._exception
  File "/Users/charles/mambaforge/envs/neurotechdevkit/lib/python3.10/site-packages/asgiref/sync.py", line 353, in main_wrap
    result = await self.awaitable(*args, **kwargs)
  File "/Users/charles/Development/neurotechdevkit/src/web/views.py", line 41, in index
    available_cts=get_available_cts(current_app.config["CT_FOLDER"]),
KeyError: 'CT_FOLDER'
```

I'm not sure how to drop into a breakpoint in the web code - if you have instructions, I'm happy to help dig into it more!